oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/pipelined/PipelinedMongoDownloadTask.java

```diff
@@ -692,7 +692,7 @@ private void download(Bson mongoQueryFilter) throws InterruptedException, Timeou
                         if (failuresStartTimestamp == null) {
                             failuresStartTimestamp = Instant.now().truncatedTo(ChronoUnit.SECONDS);
                         }
-                        LOG.warn("Connection error downloading from MongoDB.", e);
+                        LOG.info("Non-fatal connection error downloading from MongoDB: {}", e.toString());
```

Comment:
I have seen people scan for "error" (case insensitive)... What about "Non-fatal connection interruption"?

Reply:
Done.